python_modules/dagster/dagster/_core/pipes/client.py

```diff
@@ -146,21 +146,30 @@ def no_messages_debug_text(self) -> str:
 class PipesMessageReader(ABC):
     @abstractmethod
     @contextmanager
-    def read_messages(self, handler: "PipesMessageHandler") -> Iterator[PipesParams]:
+    def read_messages(
```

Comment:
fwiw read_messages also is not really an accurate name as this point. it more instigates a reading process no?